yarn.lock

```diff
@@ -0,0 +1,6637 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Not sure if this should be added. I still think that npm is the default. Therefore I would prefer to have a `package-lock.json` instead of a `yarn.lock`.

Reply:
Yeah that is understandable, I was having issues running `npm i` with resolving some of the `semantic-release` peer deps yesterday, however I just tried it again today and seems now it works: `rm -rf yarn.lock package-lock.json && npm i` ref: https://github.com/kuncevic/node-command-line-starter/commit/be76556c7ca1b51a4f581a3530c8caa538c0ce78